content/4_Gold/Tree_Euler.mdx

```diff
@@ -542,6 +563,88 @@ public class Main {
 
 </JavaSection>
 
+<PySection>
+
+```py
+import sys
+
+sys.setrecursionlimit(10**9)
+
+MAX_N = 200001
```

Comment:
0-index the nodes when reading input & making this 2 * 10 ** 5

Reply:
Done.